pkg/endpointmanager/manager_test.go

```diff
@@ -116,45 +118,40 @@ func (epSync *dummyEpSyncher) DeleteK8sCiliumEndpointSync(e *endpoint.Endpoint)
 }
 
 func (s *EndpointManagerSuite) TestLookup(c *C) {
-	ep := endpoint.NewEndpointWithState(s, s, testipcache.NewMockIPCache(), &endpoint.FakeEndpointProxy{}, testidentity.NewMockIdentityAllocator(nil), 10, endpoint.StateReady)
-	mgr := New(&dummyEpSyncher{})
 	type args struct {
 		id string
 	}
 	type want struct {
-		ep       *endpoint.Endpoint
+		ep       bool
 		err      error
 		errCheck Checker
 	}
 	tests := []struct {
-		name        string
-		setupArgs   func() args
-		setupWant   func() want
-		preTestRun  func()
-		postTestRun func()
+		name      string
+		setupArgs func() args
+		setupWant func() want
+		//preTestRun func()
```

Comment:
Why is this commented out? Do we need it at all?

Reply:
just an oversight, good catch.